cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -569,3 +569,53 @@ Feature: JWT Authenticator - Fetch signing key
     """
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  Scenario: provider-uri with untrusted self sign certificate
```

Comment:
Add test number for example:
Scenario: ONYX-8914: provider-uri with untrusted self sign certificate
Relevant for your new tests